rook/scripts/test-rook.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+LOGGING=${2:-false}
+SUCCESSES=0
+FAILURES=0
+DEBUG_OUTPUT=("")
+CONFIG_FILE="${CK8S_CONFIG_PATH}/rook/values.yaml"
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=rook/scripts/funcs.sh
+source "${here}/funcs.sh"
+
+function test_rook_help() {
+  printf "%s\n" "[Usage]: test-rook.sh <sc|wc|both> [--logging-enabled]"
+  exit "${1:-0}"
+}
+
+function test_rook() {
+  case "${1}" in
+  sc)
+    clusters="sc"
+    ;;
+  wc)
+    clusters="wc"
+    ;;
+  both)
+    clusters="sc wc"
+    ;;
+  esac
+
+  for cluster in $clusters; do
+    export KUBECONFIG="${CK8S_CONFIG_PATH}/.state/kube_config_${cluster}.yaml"
+    DEPLOYMENTS=$(kubectl -n rook-ceph get deployments.apps -o=jsonpath='{.items[*].metadata.name}' |  xargs -n 1)
+    DAEMONSETS=$(kubectl -n rook-ceph get daemonsets.apps -o=jsonpath='{.items[*].metadata.name}' |  xargs -n 1)
+    JOBS=$(kubectl -n rook-ceph get jobs.batch -o=jsonpath='{.items[*].metadata.name}' |  xargs -n 1)
+    CEPHCLUSTERS=$(kubectl -n rook-ceph get cephclusters.ceph.rook.io -ojsonpath='{.items[*].metadata.name}' |  xargs -n 1)
+
+    echo
+    echo "Testing rook-ceph in ${cluster}.."
+    echo
+
+    echo "Testing Deployments"
+    echo "===================="
+    simpleData="$(getStatus "Deployment" "rook-ceph")"
+    for deployment in $DEPLOYMENTS; do
+      testResourceExistenceFast "Deployment" "rook-ceph" "${deployment}" "${simpleData}" "${cluster}"
```

Comment:
Am I missing something or is this a bit strange? Why should we check the existance if we just got the list of deployments from the cluster? I get the rediness part but we know that it exists. Wouldn't it be better if we had an already made list of expected deployments, then if some deployment wasn't deployed for some reason it would show up as non-existent